resources/ManageApprovers.css

```diff
@@ -1,89 +1,91 @@
-.form-container {
-	text-align: left;
-	max-width: 650px;
-	background-color: #f2f2f2;
-}
+.manage-approvers {
+	.form-container {
```

Comment:
It's interesting that this works. I guess MediaWiki's ResourceLoader is passing this through as LESS, because nested selectors aren't valid in plain CSS.